### ISSUE TITLE
[Feature]: Default-Arguments

### ISSUE THREAD

**Issue by deanle8** [InsinuateProjects/TrMenu#209]
### 📚 Have you checked the Wiki

- [X] I have checked the Wiki and determined that this feature does not exist.

### 🔗 Is there a duplicate issue

- [X] I have looked through the existing issue and found no duplicate content, or a more detailed description of the existing issue.

### 🔍 Detailed description

Can you make Default-Arguments allow the use of placeholderapi??
![image](https://user-images.githubusercontent.com/77945073/180655585-f1ef689f-60db-4fbf-9a50-aff6a636fa7a.png)
![image](https://user-images.githubusercontent.com/77945073/180655593-7f5ff510-5e62-4637-ab0f-e4832a6ffb46.png)

### ⚙ Configurations

```

```

**Comment by Dirt233** [comment]
You can use the enent.open to set argue!

**Comment by 157946a** [comment]
I don't think this is a good enhancement, because most papi variables are personal. It is a bit redundant to use personal variables in parameters and then use them elsewhere, but there will be a problem that papi sets papi and the second papi does not take effect.
I don't think this is a good enhancement, because most papi variables are personal. It is a bit redundant to use personal variables in parameters and then use them elsewhere, but there will be a problem that papi sets papi and the second papi does not take effect.
